wrapanapi/systems/lenovo.py

```diff
@@ -453,3 +487,147 @@ def get_device_unique_id(device):
 
     def disconnect(self):
         self.logger.info("LenovoSystem disconnected")
+
+    def __getattr__(self, attr):
+        """
+        Tries to delegate a not found method to objects inside self._delegate
```

Comment:
https://cfme-tests.readthedocs.io/en/master/guides/dev_guide.html

Please following contributing guidelines for docblocks, as well as the Napoleon-sphinx guides:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

First line should be summary, then empty line, then additional notes/sections. `:raises` should be under `Raises:` section